### PR TITLE
get rid of the 4x3Hirez folder

### PR DESCRIPTION
**Pull request by ronie**
@BigNoid @phil65 i could use an extra pair of eyes on this one.

currently re-touched has separate xml folders for the 4x3 and 16x9 resolution.
which one is used is defined in addon.xml

the differences between the files in both folders are minimal.

it would maintenance a lot easier if i didn't have to code every change twice,
therefore i tried to come up with an easier way to provide a gui for both 4x3 and 16x9 with just 1 folder.
this is done by using the Skin.AspectRatio infolabel to differentiate between the two.
(Skin.AspectRatio returns the 'aspect' value from addon.xml)

i've tested both 16x9 and 4x3 with these changes applied and as far as i could tell everything works as expected.

please let me know if you see any potential issues here.

### CODE REVIEW

File: 16x9/includes.xml [hunk 1]
```diff
@@ -40,12 +41,42 @@
...
+		<width>1706</width>
+	</include>
+	<include name="BackgroundWidth-4x3">
+		<width>345</width>
+	</include>
+	<include name="LabelWidth-4x3">
+		<width>335</width>
+	</include>
+	<include name="ScreenWidth-4x3">
+		<width>12880</width>
```
Comment (garbear) [inline 16x9/includes.xml]: not a clue what this does, but 12880 looks like it should be 1280

Comment (ronie) [inline 16x9/includes.xml]: thx! typo indeed.

**Comment by phil65** [comment]
Only looked through code briefly, but in general I think that approach is sane. +1 from me since this really saves a lot of time. And nice trick to put the file include nodes into an include btw, didnt deal with that practice in my skinning tools yet. :)

**Comment by BigNoid** [comment]
Everything seems to be working. 
Can you explain a bit more why you include the files on startup [here](https://github.com/xbmc/skin.re-touched/blob/ditch_4x3hirez/16x9/Startup.xml#L6) and in settingscategory [here](https://github.com/xbmc/skin.re-touched/blob/ditch_4x3hirez/16x9/SettingsCategory.xml#L7). Probably a workaround to get the files included based on bool?

The log is spamming invalid include errors on my end:

```
22:47:14 T:5392 WARNING: Skin has invalid include:
22:47:14 T:7140 WARNING: Previous line repeats 19 times.
```

**Comment by ronie** [comment]
yeah, it's to include the right set of ViewFoo.xml files (for 4:3 OR 16:9)

it's done in Startup.xml so they get loaded when you start Kodi

it's done in SettingsCategory.xml so they get loaded when you switch from another skin to re-touched.
Kodi doesn't process the Startup.xml in this case (which sucks btw)

also it's done in DIalogSeekbar.xml, which is needed in case you ReloadSkin().
Kodi doesn't process the Startup.xml in this case either, nor does it load SettingsCategory.xml
unless you happen to be in one of the settings windows when you call ReloadSkin().
it does however load a few xml files, DIalogSeekbar.xml happens to be one of them.

i'll see if i can track down what is causing those warnings.

**Comment by phil65** [comment]
It doesnt work if you just do the including in Includes.xml itself?

**Comment by ronie** [comment]
nope, i tested that but you can't conditionally include xml files in includes.xml

conditionally including constants doesn't work either btw.

**Pull request closed without merging**